docs/source/building-supergraphs/jetbrains-ide-support.mdx

```diff
@@ -4,6 +4,8 @@ title: Federation support in JetBrains IDEs
 
 The [GraphQL Plugin for JetBrains](https://plugins.jetbrains.com/plugin/8097-graphql/) provides federation-specific development features, such as auto-complete for federation directives. **However**, you must _enable_ this federation support after installing the plugin. Otherwise, your IDE might display unexpected errors while you're working with a subgraph schema.
 
+## Setup
```

Comment:
Without at least one heading in the article, there's no table of contents and then the content is extra-wide 😄